app/controllers/v1/ssj/dashboard_controller.rb

```diff
@@ -25,5 +24,28 @@ def assigned_steps
     options[:include] = ['documents']
     render json: V1::Ssj::AssignedStepsSerializer.new(steps, options)
   end
+
+  def team
+    if team = current_user&.person&.ssj_team
+      render json: V1::Ssj::TeamSerializer.new(team)
+    else
+      render json: { message: "current user is not part of team"}, status: :unprocessable_entity
+    end
+  end
```

Comment:
I wonder if it might be easier to initialize people into the SSJ with an empty team so we can always assume it is there?  I guess that assumes you've always went through the ssj though.